src/core/cgroup.h

```diff
@@ -16,7 +16,7 @@ typedef struct CGroupTasksMax {
 
 #define CGROUP_TASKS_MAX_UNSET ((const CGroupTasksMax) { .value = UINT64_MAX, .scale = 0 })
 
-static inline bool cgroup_tasks_max_isset(const CGroupTasksMax *tasks_max) {
+static inline bool CGROUP_TASKS_MAX_IS_SET(const CGroupTasksMax *tasks_max) {
```

Comment:
Hmm, actually we (mainly @poettering, I guess) sometimes name inline functions in capital letters, but I do not like that. It easily confuses us that the symbol is a macro.
We should use capital symbols only for macros.
At least, there is no reason to rename an existing function like that.